tee-worker/app-libs/stf/src/trusted_call_result.rs

```diff
@@ -51,9 +60,37 @@ impl StfExecutionResult for TrustedCallResult {
 	}
 }
 
+/// For requests that mutate the IDGraph, the response result will contain at least:
+/// - `mutated_id_graph`: the mutated IDGraph, which is the subset of the whole IDGraph
+/// - `id_graph_hash`: the new IDGraph hash after the mutation
+///
+/// Currently it applies to the following DI requests:
+/// - `link_identity`
+/// - `activate_identity`
+/// - `deactivate_identity`
+/// - `set_identity_networks`
 #[derive(Encode, Decode, Clone, Debug, PartialEq, Eq)]
 pub struct LinkIdentityResult {
-	pub id_graph: AesOutput,
```

Comment:
I added the breaking change label because of this.

Reply:
Yes, thanks!